src/node_api.cc

```diff
@@ -2054,6 +2053,49 @@ napi_status napi_instanceof(napi_env e,
     return napi_set_last_error(napi_function_expected);
   }
 
+  napi_value value, key, jsRes;
+  napi_status status;
+  napi_valuetype valueType;
+
+  // Get "Symbol" from the global object
+  status = napi_get_global(e, &value);
+  if (status != napi_ok) return status;
+  status = napi_create_string_utf8(e, "Symbol", 6, &key);
+  if (status != napi_ok) return status;
+  status = napi_get_property(e, value, key, &value);
```

Comment:
Actually is this what `v8::Symbol::ForApi()` is for? I can't tell because there is no documentation and it isn't used anywhere in Node.js.